announcements/1.18.1.md

```diff
@@ -4,16 +4,24 @@ The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.18.1.
 Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
 
 ## New features & improvements
-* Android security updates for June 2023 (See [Android Security Bulletin - June 2023](tbd) for more information).
-* The Android WebView has been updated to < insert-version-along-with-link >.
+* Android security updates for June 2023 (See [Android Security Bulletin - June 2023](https://source.android.com/docs/security/bulletin/2023-06-01) for more information).
+* The Android WebView has been updated to [114.0.5735.53](https://chromereleases.googleblog.com/2023/05/early-stable-update-for-android.html).
+* Improve our emulation GL stack to deal with the case that some games provide incomplete texture and caused broken graphics output.
+* Do not send the `touch-move` event until the `touch-start` event is fired first. This avoid unexpected events sent from the client to Android container, which causes a bunch of error logs to show up during Android runtime.
 
 ## Known issues
 
 < List known issues with the current release. Include plans to fix it, if known. >
 
 ## Bug fixes
 
-< List bugs that were part of the current release. Include JIRA or launchpad IDs and links for the bugs >
+* [AC-1558](https://warthogs.atlassian.net/browse/AC-1558) Fix the crash caused by the `boot-application-relaunced` watchdog when launching Beach Buggy Racing in the portrait mode.
+* [AC-1600](https://warthogs.atlassian.net/browse/AC-1600) Fix the permission denial error when broadcasting `ACTION_CLOSE_SYSTEM_DIALOGS` intent from calendar app.
+* [AC-1606](https://warthogs.atlassian.net/browse/AC-1606) Fix the image ID out-of-sync issue when adding an image into AMS.
+* [AC-1608](https://warthogs.atlassian.net/browse/AC-1608) Fix the `InputReader: Received unexpected event` error when enabling the touch emulation by default.
```

Comment:
Isn't that the same as above?
> Do not send the `touch-move` event until the `touch-start` event is fired first. This avoid unexpected events sent from the client to Android container, which causes a bunch of error logs to show up during Android runtime.

Do we need to have it in both sections? What do you think?

Reply:
Let me drop it from the `## New features & improvements` seciton then